CHANGELOG.md

```diff
@@ -3,7 +3,25 @@
 BREAKING CHANGES:
 * Helm
   * Minimum Kubernetes version supported is 1.19 and now matches what is stated in the `README.md` file.  [[GH-1049](https://github.com/hashicorp/consul-k8s/pull/1049)] 
+* ACLs
```

Comment:
we should also mention that the old acl tokens and their k8s secrets would need to be cleaned up manually by the user after upgrading to this version.

Reply:
Oh and we also need to add as a breaking change that if you're using mesh federation with mesh gateways, you will now be required to set `federation.k8sAuthMethodHost` in the secondary datacenter:

https://github.com/hashicorp/consul-k8s/blob/acls-refactor-base-branch/charts/consul/values.yaml#L482

Reply:
This is great.  Thank you.  I just took a stab at this.  Not sure if cleaning up old tokens, just means deleting old secrets or if there is more.

Also, take a look at the ACLs section and how I added this and the other two items as breaking changes.  If you have suggestions on a better way to convey all of this or if something is missing, please let me know.